models/index.js

```diff
@@ -32,8 +32,6 @@ modelNames.forEach(modelName => {
 	}
 })
 
-sequelize.sync()
```

Comment:
Removing this is a net win. Sure, the tables can now be out-of-sync with the models, but the start-up time is faster, and we can actually do bidirectional relations, like sequelize expects.